internal/provider/utils/filtering.go

```diff
@@ -45,7 +45,7 @@ func filterServer(server models.Server,
 		return true
 	}
 
-	if *selection.StreamOnly && !server.Stream {
+	if *selection.StreamOnly && (!server.Stream || !server.Features.Streaming) {
```

Comment:
We should have a single `Stream` field, so probably just set the value of `Features.Streaming` into `Stream`.

Reply:
Will do! (optionally with a TODO for v4, depending on what you decide about the features struct)